Frameworks/CoreText/DWriteWrapper_CoreText.mm

```diff
@@ -393,14 +393,14 @@ HRESULT STDMETHODCALLTYPE GetPixelsPerDip(_In_opt_ void* clientDrawingContext, _
  *
  * @return Unmutable array of _CTLine objects created with the requested parameters.
  */
-static _CTLine* _DWriteGetLine(CFAttributedStringRef string) {
+static CTLineRef _DWriteGetLine(CFAttributedStringRef string) {
```

Comment:
I'm not sure why it was named like that too ;P


---
In reply to: [112725508](https://github.com/Microsoft/WinObjC/pull/2586#discussion_r112725508) [](ancestors = 112725508)